### PR TITLE
Implementing Battle Droid Escort

### PR DESCRIPTION
**Pull request by supercodepoet**
image

### CODE REVIEW

**Review by adbronico — APPROVED** [review]
a

File: test/server/cards/03_TWI/units/BattleDroidEscort.spec.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+                context.player1.clickCard(context.battleDroidEscort);
+                expect(context.player2).toBeActivePlayer();
+
+                let battleDroids = context.player1.findCardsByName('battle-droid');
+                expect(battleDroids.length).toBe(1);
+                expect(battleDroids).toAllBeInZone('groundArena');
+                expect(battleDroids.every((battleDroid) => battleDroid.exhausted)).toBeTrue();
+
+
```
Comment (AMMayberry1) [inline test/server/cards/03_TWI/units/BattleDroidEscort.spec.ts:28]: nit: two blank lines

Comment (supercodepoet) [reply test/server/cards/03_TWI/units/BattleDroidEscort.spec.ts]: fixed up

**Review by AMMayberry1 — DISMISSED** [review]

File: server/game/cards/03_TWI/units/BattleDroidEscort.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class BattleDroidLegion extends NonLeaderUnitCard {
```
Comment (AMMayberry1) [inline server/game/cards/03_TWI/units/BattleDroidEscort.ts:4]: The tests caught this - wrong class name

Comment (supercodepoet) [reply server/game/cards/03_TWI/units/BattleDroidEscort.ts]: ok, that is embarrassing, fixed up

**Review by AMMayberry1 — APPROVED** [review]

**Pull request merged**